velero-plugins/configmap/restore.go

```diff
@@ -0,0 +1,39 @@
+package configmap
+
+import (
+	"github.com/konveyor/openshift-velero-plugin/velero-plugins/common"
+	"github.com/sirupsen/logrus"
+	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
+	"k8s.io/apimachinery/pkg/api/meta"
+)
+
+// RestorePlugin is a restore item action plugin for Velero
+type RestorePlugin struct {
+	Log logrus.FieldLogger
+}
+
+// AppliesTo returns a velero.ResourceSelector that applies to configmaps
+func (p *RestorePlugin) AppliesTo() (velero.ResourceSelector, error) {
+	return velero.ResourceSelector{
+		IncludedResources: []string{"configmaps"},
+	}, nil
+}
+
+// Execute action for the restore plugin for the configmap resource
+// We want to exclude from backup configmaps that contain ownerReference to build pods
+func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*velero.RestoreItemActionExecuteOutput, error) {
+	p.Log.Info("[cm-restore] Entering ConfigMap restore plugin")
+	meta, err := meta.Accessor(input.Item)
+	if err != nil {
+		p.Log.Warnf("[cm-restore] Unable to accessed metadata, err: %v", err)
```

Comment:
if we have err here, then meta will be nil -- we probably need a return statement here.

Reply:
resolved in e552e6d26f785743d44ec5e453968769d7c85bf5